resources/zib/zib-LivingSituation.xml

```diff
@@ -0,0 +1,198 @@
+      <path value="Observation.code.coding" />
+      <slicing>
+        <discriminator>
+          <type value="pattern" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.code.coding:livingSituationCode">
+      <path value="Observation.code.coding" />
+      <sliceName value="livingSituationCode" />
+      <min value="1" />
+      <max value="1" />
+      <patternCoding>
+        <system value="http://snomed.info/sct" />
+        <code value="365508006" />
+      </patternCoding>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueCodeableConcept">
+      <path value="Observation.valueCodeableConcept" />
+      <sliceName value="valueCodeableConcept" />
+      <short value="HouseType" />
+      <definition value="Type of home the patient lives in." />
+      <alias value="WoningType" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.7.8.1--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-livingsituation-v3.3-2020EN" />
+        <map value="NL-CM:7.8.3" />
+        <comment value="HouseType" />
+      </mapping>
+    </element>
+    <element id="Observation.note">
+      <path value="Observation.note" />
+      <short value="Comment" />
+      <definition value="Comments in free text on problems or details of the patient’s living situation that may be relevant." />
+      <alias value="Toelichting" />
+      <max value="1" />
+      <mapping>
+        <identity value="zib-livingsituation-v3.3-2020EN" />
+        <map value="NL-CM:7.8.2" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.component:homeAdaption">
+      <path value="Observation.component" />
+      <sliceName value="homeAdaption" />
+    </element>
+    <element id="Observation.component:homeAdaption.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="118871000146108" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:homeAdaption.value[x]">
```

Comment:
Het is hier niet nodig om `.value` nogmaals te slicen, je hebt al een component slice specifiek voor dit concept. Dus je kan hier gewoon direct `.value` gebruiken. Dat geldt ook voor de `:livingCondition` slice.

Reply:
Unsliced and use the direct value[x]